src/sync/index.js

```diff
@@ -4,6 +4,41 @@ const _ = require('lodash');
 const Qweb3 = require('qweb3').default;
 const Contract = require('qweb3').Contract;
 
+
+
+const loglvl = process.env.loglvl || 'info';
+
+var winston = require('winston');
+var winston_config = winston.config;
+var logger = new (winston.Logger)({
```

Comment:
looks like this entire logic is the same for both `src/index.js` and `src/sync/index.js`. can you move into a utility method so there is no code duplication?